job-search-react/src/components/JobApp/JobApp.js

```diff
@@ -88,23 +89,22 @@ export default function JobApp(props) {
 				{/*  End Empty Grid  */}
 
 				{/*  Grid  */}
-				<div className={`jobgrid-container ${props.getDarkTheme}`}>
-					<div
-						className={`jobgrid hide-native-scrollbar ${
-							jobData && jobData.results.length > 0 ? '' : 'hidden'
-						}`}
-					>
-						{/* we need to check that jobData.data exists before we can map the results of the fetch. https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator*/}
-						{jobData &&
-							jobData.results.map((job) => (
-								<Card
-									isSubmitted={formSubmitted}
-									jobData={job}
-									theme={props.theme}
-								/>
-							))}
+
+
+					<div className={`jobgrid-container ${props.getDarkTheme}`}>
+							<ScrollContainer
```

Comment:
I can't use the mouse wheel to scroll. Please add `hideScrollbars={false}` to `<ScrollContainer`